docs/file-reference/index.md

```diff
@@ -297,6 +319,59 @@ projects:
 | spareCheckoutDir | string | What directory to use when pulling |
 
 
+# StarterProject Object
+
+> Pulling from a git location
+
+```yaml
+starterProjects:
+  - name: quarkus-quickstart
+    description: minimal CRUD app with quarkus
+    git:
+      location: https://github.com/quarkusio/quarkus-quickstarts
+      branch: master
+      sparseCheckoutDir: /getting-started/ # optional checkout dir
+      startPoint: 1.7.0.Final # tag or commit to start at
+```   
+
+
+> Pulling from a GitHub location
+
+```yaml
+projects:
+  - name: nodejs-web-app
+    description: nodejs app
+    github:
+      location: https://github.com/odo-devfiles/nodejs-ex.git
+      branch: master
+      sparseCheckoutDir: /app/ # optional checkout dir
+      startPoint: 1.0.0 # tag or commit to start at
+```
+
+> Pulling from a zip 
+
+```yaml
+projects:
+  - name: nodejs-web-app
+    description: nodejs web app
+    zip:
+      location: https://github.com/odo-devfiles/nodejs-ex/archive/0.0.1.zip
+      sparseCheckoutDir: /app/ # optional checkout dir
+```
+
+Each project may contain three different objects, `git`, `github` or `zip`.
+
+
+
+| Key       | Type                          | Description                                                                        |
+|-----------|-------------------------------|------------------------------------------------------------------------------------|
+| name      | string                        | Name of your devfile                                                               |
+| description | string                      | description of a starter project
```

Comment:
you're forgetting a `|` at the end of this

need to capitalize Description

Reply:
done.